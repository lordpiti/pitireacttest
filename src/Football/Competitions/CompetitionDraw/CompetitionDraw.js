import React, { Component } from 'react';
import apiInstance from '../../../axios-test';
import CompetitionDrawMatch from './CompetitionDrawMatch/CompetitionDrawMatch';
import './CompetitionDraw.css';

class CompetitionDraw extends Component {

  state = {
    draw: null
  }

  constructor(props) {
    super(props);

    apiInstance.get('competition/' + props.competitionData.id + '/getDraw/').then(response => {
      debugger;
      this.setState({
        draw: response.data
      });
    });
   }

  render() {

    let drawContent = null;

    if (this.state.draw) {
      drawContent = <div class="brackets_container" style={{backgroundColor: '#f0f0f0'}}>
      <table>
        <thead>
          <tr>
            <th>
                <span>Round of 16</span>
            </th>
            <th>
                <span>Quarter-finals</span>
            </th>
            <th>
                <span>Semi-finals</span>
            </th>
            <th>
                <span>final</span>
            </th>
            <th>
                <span>Semi-finals</span>
            </th>
            <th>
                <span>Quarter-finals</span>
            </th>
            <th>
                <span>Round of 16</span>
            </th>
          </tr>
        </thead>
        <tbody>
          <tr id="playground">
            <td class="round_column r_16 ">
              {this.state.draw.eightLeft.map(match =>
                <CompetitionDrawMatch match={match} competitionData={this.props.competitionData}>
                </CompetitionDrawMatch>)
              }
            </td>
            <td class="round_column r_8">
              {this.state.draw.quarterFinalsLeft.map(match =>
                <CompetitionDrawMatch match={match} competitionData={this.props.competitionData}>
                </CompetitionDrawMatch>)
              }
            </td>
            <td class="round_column r_4">
              <CompetitionDrawMatch match={this.state.draw.semifinalsLeft} competitionData={this.props.competitionData}>
              </CompetitionDrawMatch>
            </td>
            <td class="round_column r_2 final">
                <div class="winner_team">
                    <span>WINNER
                        <a href="#">
                          <img width="25" height="25" src={this.state.draw.final.goalsLocal>this.state.draw.final.goalsVisitor?this.state.draw.final.localTeam.pictureLogo.url:this.state.draw.final.visitorTeam.pictureLogo.url} alt={this.state.draw.final.localTeam.name} />
                          <span>
                              {(this.state.draw.final.goalsLocal>this.state.draw.final.goalsVisitor)?this.state.draw.final.localTeam.name:this.state.draw.final.visitorTeam.name}
                          </span>
                        </a>
                    </span>
                </div>
                <div class="mtch_container">
                    <div class="match_unit">
                        <div class="m_segment m_top winner first" data-team-id="9">
                            <span>
                                <a href="#">
                                    <img width="25" height="25" src={this.state.draw.final.localTeam.pictureLogo.url} alt={this.state.draw.final.localTeam.name} />
                                    <span class="truncate">{this.state.draw.final.localTeam.name}</span>
                                </a>
                                <strong>{this.state.draw.final.goalsLocal}</strong>
                            </span>
                        </div>
                        <div class="m_segment m_botm winner second" data-team-id="2">
                            <span>
                                <a href="#">
                                    <img width="25" height="25" src={this.state.draw.final.visitorTeam.pictureLogo.url} alt={this.state.draw.final.visitorTeam.name} />
                                    <span class="truncate">{this.state.draw.final.visitorTeam.name}</span>
                                </a>
                                <strong>{this.state.draw.final.goalsVisitor}</strong>
                            </span>
                        </div>
                        <div class="m_dtls">
                            <a routerLink="/competitions/detail/{{competitionData.id}}/match/{{draw.final.matchId}}">
                                <span>{this.state.draw.final.date}</span>
                            </a>
                        </div>
                    </div>
                </div>
            </td>
            <td class="round_column r_4 reversed">
              <CompetitionDrawMatch match={this.state.draw.semifinalsRight} competitionData={this.props.competitionData}>
              </CompetitionDrawMatch>
            </td>
            <td class="round_column r_8 reversed">
              {this.state.draw.quarterFinalsRight.map(match =>
                <CompetitionDrawMatch match={match} competitionData={this.props.competitionData}>
                </CompetitionDrawMatch>)
              }
            </td>
            <td class="round_column r_16 reversed">
              {this.state.draw.eightRight.map(match =>
                <CompetitionDrawMatch match={match} competitionData={this.props.competitionData}>
                </CompetitionDrawMatch>)
              }
            </td>
          </tr>
        </tbody>
      </table>
    </div>
    }

    return (
      <div>
        <h3>Playoff</h3>

        {drawContent}
      </div>
    )
  };

}

export default CompetitionDraw;