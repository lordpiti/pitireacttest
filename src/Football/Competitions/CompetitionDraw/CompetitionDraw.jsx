import React, { Component } from 'react';
import apiInstance from '../../utilities/axios-test';
import CompetitionDrawMatch from './CompetitionDrawMatch/CompetitionDrawMatch';
import './CompetitionDraw.css';
import Formatters from '../../utilities/formatters';
import { Link } from 'react-router-dom';

class CompetitionDraw extends Component {

  state = {
    draw: null
  }

  constructor(props) {
    super(props);

    apiInstance.get('competition/' + props.competitionData.id + '/getDraw/').then(response => {
      this.setState({
        draw: response.data
      });
    });
  }

  render() {

    let drawContent = null;

    if (this.state.draw) {
      drawContent = <div className="brackets_container" style={{ backgroundColor: '#f0f0f0' }}>
        <table>
          <thead>
            <tr>
              <th>
                <span>Round of 16</span>
              </th>
              <th>
                <span>Quarter-finals</span>
              </th>
              <th>
                <span>Semi-finals</span>
              </th>
              <th>
                <span>final</span>
              </th>
              <th>
                <span>Semi-finals</span>
              </th>
              <th>
                <span>Quarter-finals</span>
              </th>
              <th>
                <span>Round of 16</span>
              </th>
            </tr>
          </thead>
          <tbody>
            <tr id="playground">
              <td className="round_column r_16 ">
                {this.state.draw.eightLeft.map(match =>
                  <CompetitionDrawMatch key={match.matchId} match={match} competitionData={this.props.competitionData}>
                  </CompetitionDrawMatch>)
                }
              </td>
              <td className="round_column r_8">
                {this.state.draw.quarterFinalsLeft.map(match =>
                  <CompetitionDrawMatch key={match.matchId} match={match} competitionData={this.props.competitionData}>
                  </CompetitionDrawMatch>)
                }
              </td>
              <td className="round_column r_4">
                <CompetitionDrawMatch match={this.state.draw.semifinalsLeft} competitionData={this.props.competitionData}>
                </CompetitionDrawMatch>
              </td>
              <td className="round_column r_2 final">
                <div className="winner_team">
                  <span>WINNER
                    <a href="#">
                      <img width="25" height="25" src={this.state.draw.final.goalsLocal > this.state.draw.final.goalsVisitor ? this.state.draw.final.localTeam.pictureLogo.url : this.state.draw.final.visitorTeam.pictureLogo.url} alt={this.state.draw.final.localTeam.name} />
                      <span>
                        {(this.state.draw.final.goalsLocal > this.state.draw.final.goalsVisitor) ? this.state.draw.final.localTeam.name : this.state.draw.final.visitorTeam.name}
                      </span>
                    </a>
                  </span>
                </div>
                <div className="mtch_container">
                  <div className="match_unit">
                    <div className="m_segment m_top winner first" data-team-id="9">
                      <span>
                        <a href="#">
                          <img width="25" height="25" src={this.state.draw.final.localTeam.pictureLogo.url} alt={this.state.draw.final.localTeam.name} />
                          <span className="truncate">{this.state.draw.final.localTeam.name}</span>
                        </a>
                        <strong>{this.state.draw.final.goalsLocal}</strong>
                      </span>
                    </div>
                    <div className="m_segment m_botm winner second" data-team-id="2">
                      <span>
                        <a href="#">
                          <img width="25" height="25" src={this.state.draw.final.visitorTeam.pictureLogo.url} alt={this.state.draw.final.visitorTeam.name} />
                          <span className="truncate">{this.state.draw.final.visitorTeam.name}</span>
                        </a>
                        <strong>{this.state.draw.final.goalsVisitor}</strong>
                      </span>
                    </div>
                    <div className="m_dtls">
                      <Link to={{
                        pathname: `/competitions/competition-details/${this.props.competitionData.id}/competition-rounds/match/${this.state.draw.final.matchId}`
                      }}>
                        <span>{Formatters.formatDate(this.state.draw.final.date)}</span>
                      </Link>
                    </div>
                  </div>
                </div>
              </td>
              <td className="round_column r_4 reversed">
                <CompetitionDrawMatch match={this.state.draw.semifinalsRight} competitionData={this.props.competitionData}>
                </CompetitionDrawMatch>
              </td>
              <td className="round_column r_8 reversed">
                {this.state.draw.quarterFinalsRight.map(match =>
                  <CompetitionDrawMatch key={match.matchId} match={match} competitionData={this.props.competitionData}>
                  </CompetitionDrawMatch>)
                }
              </td>
              <td className="round_column r_16 reversed">
                {this.state.draw.eightRight.map(match =>
                  <CompetitionDrawMatch key={match.matchId} match={match} competitionData={this.props.competitionData}>
                  </CompetitionDrawMatch>)
                }
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    }

    return (
      <div>
        <h3>Playoff</h3>

        {drawContent}
      </div>
    )
  };

}

export default CompetitionDraw;