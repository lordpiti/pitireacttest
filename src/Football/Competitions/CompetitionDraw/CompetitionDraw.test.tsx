import React from 'react';
import CompetitionDraw from './CompetitionDraw';
import { shallow } from 'enzyme';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import mockDraw from './mockDraw.json';
import { vi } from 'vitest';

const middlewares = [thunk] as any;
const mockStore = configureMockStore(middlewares);
const testState = {
  competitions: {
    competitionList: [],
    currentCompetition: {
      drawData: mockDraw,
    },
  },
  players: { players: [], filteredPlayers: [] },
  teams: {
    teamList: [],
    currentTeam: {},
  },
  global: {
    loading: false,
    dash: {
      message: '',
      open: false,
    },
  },
};

const competitionData = { id: 1 };
let store = null;

describe('CompetitionDraw', () => {
  it(`should render correctly`, () => {
    store = mockStore(testState);
    vi.spyOn(store, 'dispatch');
    const wrapper = shallow(
      <CompetitionDraw competitionData={competitionData} store={store} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
