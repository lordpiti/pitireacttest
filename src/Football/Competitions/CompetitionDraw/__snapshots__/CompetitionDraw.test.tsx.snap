// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompetitionDraw should render correctly 1`] = `
<ContextProvider
  value={null}
>
  <CompetitionDraw
    competitionData={
      Object {
        "id": 1,
      }
    }
    draw={
      Object {
        "eightLeft": Array [
          Object {
            "date": "2009-10-26T00:00:00",
            "goalsLocal": 0,
            "goalsVisitor": 0,
            "localTeam": Object {
              "city": null,
              "id": 4,
              "name": "Valencia C.F",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 764,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 9,
              "name": "Almeria",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "1720.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/1720.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
          },
          Object {
            "date": "2009-07-02T00:00:00",
            "goalsLocal": 1,
            "goalsVisitor": 0,
            "localTeam": Object {
              "city": null,
              "id": 7,
              "name": "Málaga",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "5915d799-b58a-4ef6-87a0-cf45ed2cbe87-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/5915d799-b58a-4ef6-87a0-cf45ed2cbe87-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 767,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 14,
              "name": "Osasuna",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "46be640c-1bdc-4472-a6dc-ebfc5df9687a-211.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/46be640c-1bdc-4472-a6dc-ebfc5df9687a-211.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
          },
          Object {
            "date": "2009-06-02T00:00:00",
            "goalsLocal": 3,
            "goalsVisitor": 1,
            "localTeam": Object {
              "city": null,
              "id": 6,
              "name": "Atlético de Madrid",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 766,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 11,
              "name": "Sporting de Gijón",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "df217d75-e06a-4b7f-8089-4c49eb307c0d-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/df217d75-e06a-4b7f-8089-4c49eb307c0d-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
          },
          Object {
            "date": "2009-07-22T00:00:00",
            "goalsLocal": 2,
            "goalsVisitor": 1,
            "localTeam": Object {
              "city": null,
              "id": 8,
              "name": "Athletic de Bilbao",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "749.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/749.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 768,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 16,
              "name": "Sevilla C.F",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "ce743178-43a7-47a0-b192-94caf6216695-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/ce743178-43a7-47a0-b192-94caf6216695-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
          },
        ],
        "eightRight": Array [
          Object {
            "date": "2009-05-18T00:00:00",
            "goalsLocal": 3,
            "goalsVisitor": 0,
            "localTeam": Object {
              "city": null,
              "id": 5,
              "name": "Villarreal",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "987.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/987.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 765,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 13,
              "name": "Racing de Santander",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "b3ab87b1-5d7a-4c52-a7c7-71f33c90f3c9-1473.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/b3ab87b1-5d7a-4c52-a7c7-71f33c90f3c9-1473.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
          },
          Object {
            "date": "2009-10-22T00:00:00",
            "goalsLocal": 1,
            "goalsVisitor": 1,
            "localTeam": Object {
              "city": null,
              "id": 1,
              "name": "R.C Deportivo",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "f20b9256-8fb6-4d62-a177-286fc2e18e69-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/f20b9256-8fb6-4d62-a177-286fc2e18e69-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 761,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 12,
              "name": "Numancia",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "1952.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/1952.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
          },
          Object {
            "date": "2009-06-26T00:00:00",
            "goalsLocal": 0,
            "goalsVisitor": 3,
            "localTeam": Object {
              "city": null,
              "id": 2,
              "name": "F.C Barcelona",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "07f2de74-b684-421b-a508-9bde3496aeed-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/07f2de74-b684-421b-a508-9bde3496aeed-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 762,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 10,
              "name": "Valladolid",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "849.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/849.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
          },
          Object {
            "date": "2009-07-09T00:00:00",
            "goalsLocal": 1,
            "goalsVisitor": 0,
            "localTeam": Object {
              "city": null,
              "id": 3,
              "name": "Real Madrid",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "b111ec0e-3c8a-41e2-82ae-d7a8c8834a4a-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/b111ec0e-3c8a-41e2-82ae-d7a8c8834a4a-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 763,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 15,
              "name": "Espanyol",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
          },
        ],
        "final": Object {
          "date": "2009-05-20T00:00:00",
          "goalsLocal": 0,
          "goalsVisitor": 2,
          "localTeam": Object {
            "city": null,
            "id": 4,
            "name": "Valencia C.F",
            "pictureLogo": Object {
              "base64String": null,
              "bytes": null,
              "containerReference": "mycontainer",
              "fileName": "ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
              "id": 0,
              "url": "https://pititest.blob.core.windows.net/mycontainer/ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
            },
            "playerList": null,
            "stadium": null,
          },
          "matchId": 775,
          "stadium": Object {
            "address": null,
            "capacity": 0,
            "id": 1,
            "name": "Estadio de Riazor",
          },
          "visitorTeam": Object {
            "city": null,
            "id": 5,
            "name": "Villarreal",
            "pictureLogo": Object {
              "base64String": null,
              "bytes": null,
              "containerReference": "mycontainer",
              "fileName": "987.jpg",
              "id": 0,
              "url": "https://pititest.blob.core.windows.net/mycontainer/987.jpg",
            },
            "playerList": null,
            "stadium": null,
          },
        },
        "quarterFinalsLeft": Array [
          Object {
            "date": "2009-04-20T00:00:00",
            "goalsLocal": 0,
            "goalsVisitor": 2,
            "localTeam": Object {
              "city": null,
              "id": 4,
              "name": "Valencia C.F",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 770,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 14,
              "name": "Osasuna",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "46be640c-1bdc-4472-a6dc-ebfc5df9687a-211.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/46be640c-1bdc-4472-a6dc-ebfc5df9687a-211.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
          },
          Object {
            "date": "2009-04-23T00:00:00",
            "goalsLocal": 2,
            "goalsVisitor": 2,
            "localTeam": Object {
              "city": null,
              "id": 6,
              "name": "Atlético de Madrid",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 772,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 16,
              "name": "Sevilla C.F",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "ce743178-43a7-47a0-b192-94caf6216695-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/ce743178-43a7-47a0-b192-94caf6216695-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
          },
        ],
        "quarterFinalsRight": Array [
          Object {
            "date": "2009-03-12T00:00:00",
            "goalsLocal": 2,
            "goalsVisitor": 1,
            "localTeam": Object {
              "city": null,
              "id": 5,
              "name": "Villarreal",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "987.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/987.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 771,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 12,
              "name": "Numancia",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "1952.jpg",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/1952.jpg",
              },
              "playerList": null,
              "stadium": null,
            },
          },
          Object {
            "date": "2009-08-11T00:00:00",
            "goalsLocal": 1,
            "goalsVisitor": 1,
            "localTeam": Object {
              "city": null,
              "id": 2,
              "name": "F.C Barcelona",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "07f2de74-b684-421b-a508-9bde3496aeed-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/07f2de74-b684-421b-a508-9bde3496aeed-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
            "matchId": 769,
            "stadium": Object {
              "address": null,
              "capacity": 0,
              "id": 1,
              "name": "Estadio de Riazor",
            },
            "visitorTeam": Object {
              "city": null,
              "id": 15,
              "name": "Espanyol",
              "pictureLogo": Object {
                "base64String": null,
                "bytes": null,
                "containerReference": "mycontainer",
                "fileName": "afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
                "id": 0,
                "url": "https://pititest.blob.core.windows.net/mycontainer/afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
              },
              "playerList": null,
              "stadium": null,
            },
          },
        ],
        "semifinalsLeft": Object {
          "date": "2009-08-13T00:00:00",
          "goalsLocal": 3,
          "goalsVisitor": 3,
          "localTeam": Object {
            "city": null,
            "id": 4,
            "name": "Valencia C.F",
            "pictureLogo": Object {
              "base64String": null,
              "bytes": null,
              "containerReference": "mycontainer",
              "fileName": "ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
              "id": 0,
              "url": "https://pititest.blob.core.windows.net/mycontainer/ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
            },
            "playerList": null,
            "stadium": null,
          },
          "matchId": 773,
          "stadium": Object {
            "address": null,
            "capacity": 0,
            "id": 1,
            "name": "Estadio de Riazor",
          },
          "visitorTeam": Object {
            "city": null,
            "id": 6,
            "name": "Atlético de Madrid",
            "pictureLogo": Object {
              "base64String": null,
              "bytes": null,
              "containerReference": "mycontainer",
              "fileName": "2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
              "id": 0,
              "url": "https://pititest.blob.core.windows.net/mycontainer/2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
            },
            "playerList": null,
            "stadium": null,
          },
        },
        "semifinalsRight": Object {
          "date": "2009-02-24T00:00:00",
          "goalsLocal": 0,
          "goalsVisitor": 2,
          "localTeam": Object {
            "city": null,
            "id": 5,
            "name": "Villarreal",
            "pictureLogo": Object {
              "base64String": null,
              "bytes": null,
              "containerReference": "mycontainer",
              "fileName": "987.jpg",
              "id": 0,
              "url": "https://pititest.blob.core.windows.net/mycontainer/987.jpg",
            },
            "playerList": null,
            "stadium": null,
          },
          "matchId": 774,
          "stadium": Object {
            "address": null,
            "capacity": 0,
            "id": 1,
            "name": "Estadio de Riazor",
          },
          "visitorTeam": Object {
            "city": null,
            "id": 15,
            "name": "Espanyol",
            "pictureLogo": Object {
              "base64String": null,
              "bytes": null,
              "containerReference": "mycontainer",
              "fileName": "afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
              "id": 0,
              "url": "https://pititest.blob.core.windows.net/mycontainer/afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
            },
            "playerList": null,
            "stadium": null,
          },
        },
      }
    }
    loadDrawData={[Function]}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  />
</ContextProvider>
`;
